### ISSUE TITLE
Transaction Dialog additional buttons

### ISSUE THREAD

**Issue by vomikan** [moneymanagerex/moneymanagerex#4607]
I suggest adding additional buttons.

![image](https://user-images.githubusercontent.com/6836805/173349064-ab2f86cd-ca64-4982-966b-74d549f6617d.png)

https://www.svgrepo.com/svg/25604/calculator

https://github.com/dlasalle/wx-calc

**Comment by whalley** [comment]
Are these buttons just for image purposes or are you expecting them to bring up new dialogs/popups?

**Comment by vomikan** [comment]
Yes, these are buttons that trigger dialog boxes.

**Comment by gabriele-v** [comment]
Why we would need a calculator if it's already embedded in the amount field? Or we remove it from there?

**Comment by whalley** [comment]
> Why we would need a calculator if it's already embedded in the amount field? Or we remove it from there?

Some people like the visual representation of a calculator.

**Comment by spillaip** [comment]
I completely agree! User experience (UX) is absolutely crucial for attracting and retaining a user base.  Adding features like one-click buttons for payee, category, and calculator definitely enhances UX in a positive way. It makes the process smoother, faster, and more intuitive for users.

Here's a breakdown of the benefits:

Reduced steps: By offering buttons for these functions, users can access them directly instead of navigating through menus, saving time and effort.
Increased efficiency: Easy access to these features streamlines the process for users, allowing them to complete tasks quicker.
Improved clarity: Clear buttons communicate the functionality effectively, making it easier for users to understand what each button does.
However, it's important to strike a balance:

Prioritize existing features: Make sure the new buttons don't clutter the interface and hinder access to existing functionalities.
Maintain consistency: Keep the design of the buttons consistent with the overall visual style of the application.
By implementing these buttons thoughtfully, you can significantly improve the user experience and make your application more user-friendly, which can lead to a larger and more engaged user base.
